test/refactoredTestSetup.ts

```diff
@@ -0,0 +1,336 @@
+/*
```

Comment:
@mingxuanzhangsfdx Would you mind taking the accumulated changes to this file and apply them to TestSetup and then remove the new file. Tracking changes across two different files is a pain.

Reply:
When I started the effort, I wanted to make sure my change did not affect the tests that I have not taken care of because there are a lot tests left. Sorry for the confusion.